frontend/src/components/TypeaheadSelect.tsx

```diff
@@ -33,6 +34,11 @@ export interface TypeaheadSelectOption extends Omit<SelectOptionProps, 'content'
   /** Indicator for option being selected */
   isSelected?: boolean;
   dropdownLabel?: React.ReactNode;
+  selectedLabel?: React.ReactNode;
+  group?: {
+    groupName: string;
+    groupLabel: React.ReactNode;
+  };
```

Comment:
just make it a single groupLabel string

Reply:
i tested it and that is the expected group label from patternfly

![Screenshot 2025-06-06 at 12 55 44 PM](https://github.com/user-attachments/assets/34bc323e-0451-413b-9261-61dafa59c4b9)

Reply:
also there is a divider when there is only grouped ones and no non grouped ones. making the last divider unnecessary

Reply:
Could you please elaborate? 
Do you mean it _should_ look like one of the options or the default is already supposed to look like what's on the screenshot?

Reply:
the screenshot i provided is using just the name and not wrapping it in an option. that is how it works in patternfly docs and that is how we should follow it

Reply:
@Gkrumbach07 updated